Blockchain/Sources/Blockchain/VMInvocations/HostCall/HostCalls.swift

```diff
@@ -428,7 +443,8 @@ public class Info: HostCall {
         } else if value == nil {
             state.writeRegister(Registers.Index(raw: 7), HostCallResultCode.NONE.rawValue)
         } else {
-            state.writeRegister(Registers.Index(raw: 7), HostCallResultCode.OK.rawValue)
+            state.writeRegister(Registers.Index(raw: 7), value!.count)
```

Comment:
```suggestion
        } else if let valuel {
            state.writeRegister(Registers.Index(raw: 7), value.count)
        } else {
            state.writeRegister(Registers.Index(raw: 7), HostCallResultCode.NONE.rawValue)
```

so we can avoid the force unwrap

Reply:
will be included in the next pr